Tests fail after pulling updated changes

vulnscan git:(master) go test ./...
ok  	github.com/simplycubed/vulnscan	(cached)
?   	github.com/simplycubed/vulnscan/adapters	[no test files]
--- FAIL: TestConfigurationAdapterFromPath (0.00s)
    configuration_test.go:96:
        	Error Trace:	configuration_test.go:96
        	Error:      	Received unexpected error:
        	            	open /Users/charlesgreen/go/src/github.com/simplycubed/vulnscan/test/data/configuration/vulnscan.toml: no such file or directory
        	Test:       	TestConfigurationAdapterFromPath
    output_mocks.go:27: INFO| : searching for configuration files on /Users/charlesgreen/go/src/github.com/simplycubed/vulnscan/test/data/configuration/vulnscan.toml, /Users/charlesgreen/go/src/github.com/simplycubed/vulnscan/vulnscan.toml, /Users/charlesgreen/go/src/github.com/simplycubed/vulnscan/vulnscan.yaml, /Users/charlesgreen/go/src/github.com/simplycubed/vulnscan/vulnscan.json, /var/folders/qh/sb086bgn07b5k7sjzp_br1tc0000gn/T/go-build228065562/b155/vulnscan.toml, /var/folders/qh/sb086bgn07b5k7sjzp_br1tc0000gn/T/go-build228065562/b155/vulnscan.yaml, /var/folders/qh/sb086bgn07b5k7sjzp_br1tc0000gn/T/go-build228065562/b155/vulnscan.json
    output_mocks.go:27: WARN| : configuration file not found, using default configuration
    configuration_test.go:79:
        	Error Trace:	configuration_test.go:79
        	            				configuration_test.go:97
        	Error:      	Not equal:
        	            	expected: map[entities.AnalysisCheck]bool{"DoAnalysis":true, "DoCode":true, "DoPList":true}
        	            	actual  : map[entities.AnalysisCheck]bool{"DoAnalysis":true, "DoCode":true, "DoFiles":true, "DoPList":true, "DoStore":true}

        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1,5 +1,7 @@
        	            	-(map[entities.AnalysisCheck]bool) (len=3) {
        	            	+(map[entities.AnalysisCheck]bool) (len=5) {
        	            	  (entities.AnalysisCheck) (len=10) "DoAnalysis": (bool) true,
        	            	  (entities.AnalysisCheck) (len=6) "DoCode": (bool) true,
        	            	- (entities.AnalysisCheck) (len=7) "DoPList": (bool) true
        	            	+ (entities.AnalysisCheck) (len=7) "DoFiles": (bool) true,
        	            	+ (entities.AnalysisCheck) (len=7) "DoPList": (bool) true,
        	            	+ (entities.AnalysisCheck) (len=7) "DoStore": (bool) true
        	            	 }
        	Test:       	TestConfigurationAdapterFromPath
    configuration_test.go:84:
        	Error Trace:	configuration_test.go:84
        	            				configuration_test.go:97
        	Error:      	Not equal:
        	            	expected: "test_files/apps/source.zip"
        	            	actual  : ""

        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1 +1 @@
        	            	-test_files/apps/source.zip
        	            	+
        	Test:       	TestConfigurationAdapterFromPath
    configuration_test.go:85:
        	Error Trace:	configuration_test.go:85
        	            				configuration_test.go:97
        	Error:      	Not equal:
        	            	expected: true
        	            	actual  : false
        	Test:       	TestConfigurationAdapterFromPath
    configuration_test.go:86:
        	Error Trace:	configuration_test.go:86
        	            				configuration_test.go:97
        	Error:      	Not equal:
        	            	expected: "virus_scan_password"
        	            	actual  : ""

        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1 +1 @@
        	            	-virus_scan_password
        	            	+
        	Test:       	TestConfigurationAdapterFromPath
    configuration_test.go:87:
        	Error Trace:	configuration_test.go:87
        	            				configuration_test.go:97
        	Error:      	Not equal:
        	            	expected: "es"
        	            	actual  : "us"

        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1 +1 @@
        	            	-es
        	            	+us
        	Test:       	TestConfigurationAdapterFromPath
    configuration_test.go:88:
        	Error Trace:	configuration_test.go:88
        	            				configuration_test.go:97
        	Error:      	Expected value not to be nil.
        	Test:       	TestConfigurationAdapterFromPath
    configuration_test.go:89:
        	Error Trace:	configuration_test.go:89
        	            				configuration_test.go:97
        	Error:      	Expected value not to be nil.
        	Test:       	TestConfigurationAdapterFromPath
    configuration_test.go:100:
        	Error Trace:	configuration_test.go:100
        	Error:      	Received unexpected error:
        	            	remove /Users/charlesgreen/go/src/github.com/simplycubed/vulnscan/test/data/configuration/vulnscan.toml: no such file or directory
        	Test:       	TestConfigurationAdapterFromPath
    configuration_test.go:96:
        	Error Trace:	configuration_test.go:96
        	Error:      	Received unexpected error:
        	            	open /Users/charlesgreen/go/src/github.com/simplycubed/vulnscan/test/data/configuration/vulnscan.json: no such file or directory
        	Test:       	TestConfigurationAdapterFromPath
    output_mocks.go:27: INFO| : searching for configuration files on /Users/charlesgreen/go/src/github.com/simplycubed/vulnscan/test/data/configuration/vulnscan.json, /Users/charlesgreen/go/src/github.com/simplycubed/vulnscan/vulnscan.toml, /Users/charlesgreen/go/src/github.com/simplycubed/vulnscan/vulnscan.yaml, /Users/charlesgreen/go/src/github.com/simplycubed/vulnscan/vulnscan.json, /var/folders/qh/sb086bgn07b5k7sjzp_br1tc0000gn/T/go-build228065562/b155/vulnscan.toml, /var/folders/qh/sb086bgn07b5k7sjzp_br1tc0000gn/T/go-build228065562/b155/vulnscan.yaml, /var/folders/qh/sb086bgn07b5k7sjzp_br1tc0000gn/T/go-build228065562/b155/vulnscan.json
    output_mocks.go:27: WARN| : configuration file not found, using default configuration
    configuration_test.go:79:
        	Error Trace:	configuration_test.go:79
        	            				configuration_test.go:97
        	Error:      	Not equal:
        	            	expected: map[entities.AnalysisCheck]bool{"DoAnalysis":true, "DoCode":true, "DoPList":true}
        	            	actual  : map[entities.AnalysisCheck]bool{"DoAnalysis":true, "DoCode":true, "DoFiles":true, "DoPList":true, "DoStore":true}

        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1,5 +1,7 @@
        	            	-(map[entities.AnalysisCheck]bool) (len=3) {
        	            	+(map[entities.AnalysisCheck]bool) (len=5) {
        	            	  (entities.AnalysisCheck) (len=10) "DoAnalysis": (bool) true,
        	            	  (entities.AnalysisCheck) (len=6) "DoCode": (bool) true,
        	            	- (entities.AnalysisCheck) (len=7) "DoPList": (bool) true
        	            	+ (entities.AnalysisCheck) (len=7) "DoFiles": (bool) true,
        	            	+ (entities.AnalysisCheck) (len=7) "DoPList": (bool) true,
        	            	+ (entities.AnalysisCheck) (len=7) "DoStore": (bool) true
        	            	 }
        	Test:       	TestConfigurationAdapterFromPath
    configuration_test.go:84:
        	Error Trace:	configuration_test.go:84
        	            				configuration_test.go:97
        	Error:      	Not equal:
        	            	expected: "test_files/apps/source.zip"
        	            	actual  : ""

        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1 +1 @@
        	            	-test_files/apps/source.zip
        	            	+
        	Test:       	TestConfigurationAdapterFromPath
    configuration_test.go:85:
        	Error Trace:	configuration_test.go:85
        	            				configuration_test.go:97
        	Error:      	Not equal:
        	            	expected: true
        	            	actual  : false
        	Test:       	TestConfigurationAdapterFromPath
    configuration_test.go:86:
        	Error Trace:	configuration_test.go:86
        	            				configuration_test.go:97
        	Error:      	Not equal:
        	            	expected: "virus_scan_password"
        	            	actual  : ""

        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1 +1 @@
        	            	-virus_scan_password
        	            	+
        	Test:       	TestConfigurationAdapterFromPath
    configuration_test.go:87:
        	Error Trace:	configuration_test.go:87
        	            				configuration_test.go:97
        	Error:      	Not equal:
        	            	expected: "es"
        	            	actual  : "us"

        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1 +1 @@
        	            	-es
        	            	+us
        	Test:       	TestConfigurationAdapterFromPath
    configuration_test.go:88:
        	Error Trace:	configuration_test.go:88
        	            				configuration_test.go:97
        	Error:      	Expected value not to be nil.
        	Test:       	TestConfigurationAdapterFromPath
    configuration_test.go:89:
        	Error Trace:	configuration_test.go:89
        	            				configuration_test.go:97
        	Error:      	Expected value not to be nil.
        	Test:       	TestConfigurationAdapterFromPath
    configuration_test.go:100:
        	Error Trace:	configuration_test.go:100
        	Error:      	Received unexpected error:
        	            	remove /Users/charlesgreen/go/src/github.com/simplycubed/vulnscan/test/data/configuration/vulnscan.json: no such file or directory
        	Test:       	TestConfigurationAdapterFromPath
    configuration_test.go:96:
        	Error Trace:	configuration_test.go:96
        	Error:      	Received unexpected error:
        	            	open /Users/charlesgreen/go/src/github.com/simplycubed/vulnscan/test/data/configuration/vulnscan.yaml: no such file or directory
        	Test:       	TestConfigurationAdapterFromPath
    output_mocks.go:27: INFO| : searching for configuration files on /Users/charlesgreen/go/src/github.com/simplycubed/vulnscan/test/data/configuration/vulnscan.yaml, /Users/charlesgreen/go/src/github.com/simplycubed/vulnscan/vulnscan.toml, /Users/charlesgreen/go/src/github.com/simplycubed/vulnscan/vulnscan.yaml, /Users/charlesgreen/go/src/github.com/simplycubed/vulnscan/vulnscan.json, /var/folders/qh/sb086bgn07b5k7sjzp_br1tc0000gn/T/go-build228065562/b155/vulnscan.toml, /var/folders/qh/sb086bgn07b5k7sjzp_br1tc0000gn/T/go-build228065562/b155/vulnscan.yaml, /var/folders/qh/sb086bgn07b5k7sjzp_br1tc0000gn/T/go-build228065562/b155/vulnscan.json
    output_mocks.go:27: WARN| : configuration file not found, using default configuration
    configuration_test.go:79:
        	Error Trace:	configuration_test.go:79
        	            				configuration_test.go:97
        	Error:      	Not equal:
        	            	expected: map[entities.AnalysisCheck]bool{"DoAnalysis":true, "DoCode":true, "DoPList":true}
        	            	actual  : map[entities.AnalysisCheck]bool{"DoAnalysis":true, "DoCode":true, "DoFiles":true, "DoPList":true, "DoStore":true}

        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1,5 +1,7 @@
        	            	-(map[entities.AnalysisCheck]bool) (len=3) {
        	            	+(map[entities.AnalysisCheck]bool) (len=5) {
        	            	  (entities.AnalysisCheck) (len=10) "DoAnalysis": (bool) true,
        	            	  (entities.AnalysisCheck) (len=6) "DoCode": (bool) true,
        	            	- (entities.AnalysisCheck) (len=7) "DoPList": (bool) true
        	            	+ (entities.AnalysisCheck) (len=7) "DoFiles": (bool) true,
        	            	+ (entities.AnalysisCheck) (len=7) "DoPList": (bool) true,
        	            	+ (entities.AnalysisCheck) (len=7) "DoStore": (bool) true
        	            	 }
        	Test:       	TestConfigurationAdapterFromPath
    configuration_test.go:84:
        	Error Trace:	configuration_test.go:84
        	            				configuration_test.go:97
        	Error:      	Not equal:
        	            	expected: "test_files/apps/source.zip"
        	            	actual  : ""

        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1 +1 @@
        	            	-test_files/apps/source.zip
        	            	+
        	Test:       	TestConfigurationAdapterFromPath
    configuration_test.go:85:
        	Error Trace:	configuration_test.go:85
        	            				configuration_test.go:97
        	Error:      	Not equal:
        	            	expected: true
        	            	actual  : false
        	Test:       	TestConfigurationAdapterFromPath
    configuration_test.go:86:
        	Error Trace:	configuration_test.go:86
        	            				configuration_test.go:97
        	Error:      	Not equal:
        	            	expected: "virus_scan_password"
        	            	actual  : ""

        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1 +1 @@
        	            	-virus_scan_password
        	            	+
        	Test:       	TestConfigurationAdapterFromPath
    configuration_test.go:87:
        	Error Trace:	configuration_test.go:87
        	            				configuration_test.go:97
        	Error:      	Not equal:
        	            	expected: "es"
        	            	actual  : "us"

        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1 +1 @@
        	            	-es
        	            	+us
        	Test:       	TestConfigurationAdapterFromPath
    configuration_test.go:88:
        	Error Trace:	configuration_test.go:88
        	            				configuration_test.go:97
        	Error:      	Expected value not to be nil.
        	Test:       	TestConfigurationAdapterFromPath
    configuration_test.go:89:
        	Error Trace:	configuration_test.go:89
        	            				configuration_test.go:97
        	Error:      	Expected value not to be nil.
        	Test:       	TestConfigurationAdapterFromPath
    configuration_test.go:100:
        	Error Trace:	configuration_test.go:100
        	Error:      	Received unexpected error:
        	            	remove /Users/charlesgreen/go/src/github.com/simplycubed/vulnscan/test/data/configuration/vulnscan.yaml: no such file or directory
        	Test:       	TestConfigurationAdapterFromPath
FAIL
FAIL	github.com/simplycubed/vulnscan/adapters/input	0.028s
?   	github.com/simplycubed/vulnscan/adapters/mocks	[no test files]
ok  	github.com/simplycubed/vulnscan/adapters/output	(cached)
ok  	github.com/simplycubed/vulnscan/adapters/services	(cached)
ok  	github.com/simplycubed/vulnscan/adapters/tools	(cached)
ok  	github.com/simplycubed/vulnscan/entities	(cached)
--- FAIL: TestNormalize (4.23s)
    filesystem_test.go:64: 1: lstat /Users/charlesgreen/go/src/github.com/simplycubed/vulnscan/test/data/framework/filesystem/iVim.app: no such file or directory
--- FAIL: TestExtractBinPath (0.00s)
    filesystem_test.go:72:
        	Error Trace:	filesystem_test.go:72
        	Error:      	Received unexpected error:
        	            	.app file not found in parent folder /Users/charlesgreen/go/src/github.com/simplycubed/vulnscan/test/data/framework/filesystem
        	Test:       	TestExtractBinPath
    filesystem_test.go:73:
        	Error Trace:	filesystem_test.go:73
        	Error:      	Not equal:
        	            	expected: "/Users/charlesgreen/go/src/github.com/simplycubed/vulnscan/test/data/framework/filesystem/iVim.app/iVim"
        	            	actual  : "/Users/charlesgreen/go/src/github.com/simplycubed/vulnscan/test/data/framework/filesystem"

        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1 +1 @@
        	            	-/Users/charlesgreen/go/src/github.com/simplycubed/vulnscan/test/data/framework/filesystem/iVim.app/iVim
        	            	+/Users/charlesgreen/go/src/github.com/simplycubed/vulnscan/test/data/framework/filesystem
        	Test:       	TestExtractBinPath
    filesystem_test.go:74:
        	Error Trace:	filesystem_test.go:74
        	Error:      	Not equal:
        	            	expected: "iVim"
        	            	actual  : ""

        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1 +1 @@
        	            	-iVim
        	            	+
        	Test:       	TestExtractBinPath
FAIL
FAIL	github.com/simplycubed/vulnscan/framework	4.305s
?   	github.com/simplycubed/vulnscan/test	[no test files]
ok  	github.com/simplycubed/vulnscan/usecases/binary	(cached)
ok  	github.com/simplycubed/vulnscan/usecases/code	(cached)
ok  	github.com/simplycubed/vulnscan/usecases/files	(cached)
ok  	github.com/simplycubed/vulnscan/usecases/plist	(cached)
ok  	github.com/simplycubed/vulnscan/usecases/static	(cached)
ok  	github.com/simplycubed/vulnscan/usecases/store	(cached)
➜  vulnscan git:(master)

--- a/adapters/input/configuration_test.go
+++ b/adapters/input/configuration_test.go
@@ -90,6 +90,9 @@ func testConfigurationHelper(p string, t *testing.T) {
 }
 
 func TestConfigurationAdapterFromPath(t *testing.T) {
+	f, e := test.FindTest("configuration")
+	assert.NoError(t, e)
+	assert.NoError(t, os.MkdirAll(f, os.ModePerm))
 	for ext, content := range map[string]string{"toml": toml, "json": json, "yaml": yaml} {
 		p, e := test.FindTest("configuration", "vulnscan."+ext)
 		assert.NoError(t, e)

